docs/bootableusb.md

```diff
@@ -6,17 +6,18 @@ nav_order: 25
 
 ## Making a Bootable USB
 
+{: .warning }
+Flashing Ventoy or ISO's to an external drive will **wipe all data** on said drive. Back up anything if you have to.
+
 ### Prerequisites
 1. Another PC or laptop with at least 8GB storage
 2. A internet connection
-3. A USB drive or SD card thats at least 8 gigabytes.
-      - You will be erasing this drive, backup anything if you need to do so.
-
+3. A external drive like a USB drive or SD card thats at least 8 gigabytes.
 
 ### Downloading a ISO
 1. Determine what OS you want.
    - For Linux, keep in mind Ubuntu and any derivatives will not work.
```

Comment:
not entirely true, we need more consistency about that throughout the entire docs

Reply:
What about changing it to 4gb for Linux, 8gb for Windows?

Reply:
smh github, that comment was supposed to be on the line for ubuntu derivatives not being supported

Reply:
for clarification, what is the state of ubuntu for future commits?

Reply:
after running the audio script, they should work.

Reply:
can this be done on a separate pr?

Reply:
sure